etc/profile-a-l/bzr.profile

```diff
@@ -0,0 +1,10 @@
+# Firejail profile alias for bzr
+# This file is overwritten after every install/update
+# Persistent local customizations
+include bzr.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+# Redirect
+include brz.profile
```

Comment:
`bzr` is apparently the same program name used by bazaar itself, so it doesn't
seem like a good idea to redirect to the profile of a different program.

I'm not familiar with bazaar, so I'd probably just remove this profile.

Reply:
> bzr is apparently the same program name used by bazaar itself, so it doesn't
seem like a good idea to redirect to the profile of a different program.

That was the point of the recent breezy introduction on Arch Linux. breezy is the [drop-in replacement](https://gitlab.archlinux.org/archlinux/packaging/packages/pacman/-/commit/c68a4e6602e3488fa093a18d35202c76a730faf6) for bazaar since pacman 6.0.2-8. I've tested it, no breakage whatsoever. And it doesn't get auto-enabled via firecfg. IMO we can keep it.